src/button/tests/functional/Button.ts

```diff
@@ -0,0 +1,73 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+
+function getPage(remote: any) {
+	return remote
+		.get('http://localhost:9000/_build/common/example/?module=button')
+		.setFindTimeout(5000);
+}
+
+registerSuite({
+	name: 'Button',
+
+	'button should be visible'(this: any) {
+		return getPage(this.remote)
+			.findByCssSelector('button:first-of-type')
```

Comment:
Please change all `findByCssSelector` calls to use the generate `css-module` class names by importing the appropriate `m.css` file.